deploy/olm-catalog/kogito-operator/2.0.0-snapshot/kogito-operator.v2.0.0-snapshot.clusterserviceversion.yaml

```diff
@@ -697,5 +697,5 @@ spec:
   maturity: alpha
   provider:
     name: Red Hat
-  replaces: kogito-operator.v0.13.0
-  version: 1.0.0-snapshot
+  replaces: kogito-operator.v0.17.0
```

Comment:
Well, it is going to replace 1.0.0 in the future.

Reply:
yep but not existing yet and so pipeline is failing with 1.0.0 ;)

Reply:
oh, ok